util/execdetails/execdetails.go

```diff
@@ -381,13 +383,19 @@ func (e *RuntimeStats) Record(d time.Duration, rowNum int) {
 	atomic.AddInt32(&e.loop, 1)
 	atomic.AddInt64(&e.consume, int64(d))
 	atomic.AddInt64(&e.rows, int64(rowNum))
+	atomic.AddInt64(&e.concurrency, 0)
```

Comment:
Is `1` better than `0` in this case?

Reply:
In the original version, 0 means non-parallel execution, such as https://github.com/pingcap/tidb/blob/master/executor/projection.go#L88 . I'm not sure which is better.